intuita-webview/src/fileExplorer/App.tsx

```diff
@@ -12,11 +12,10 @@ import SearchBar from '../shared/SearchBar';
 import ActionsHeader from './ActionsHeader';
 import { vscode } from '../shared/utilities/vscode';
 
-type MainViews = Extract<View, { viewId: 'treeView' }>;
+type MainViews = Extract<View, { viewId: 'fileExplorer' }>;
```

Comment:
you can call this variable `ViewProps` and type it as `Extract<View, { viewId: 'fileExplorer' }>['viewProps']`